### ISSUE TITLE
DOWNLOAD APPLIANCE 4.0 · Virtual Philip 

### ISSUE THREAD

**Issue by slacknet** [theqvd/theqvd#70]
Hi. I have download this version and installed it on the Virtual Box by importing the configuration. I manage to login into the server with user and passwd, qvd. Now, in order to make it work I need to get root access, but I don't knwo the root user and password. Can you please hep me? I am trying to implement this service into my business as a reseller and I need to know how it works.

**Comment by slacknet** [comment]
I just want to try the demo to see how it goes

**Comment by frantracer** [comment]
Hello,

I will sum up the credentials for the QVD Appliance 4.0.0:

- Server login (as you said):
user: qvd
password: qvd

- Server root:
user: root
password: root

- Web Administration Tool (https://localhost:7443)
user: superadmin@*
password: superadmin

- QVD Client:
user: qvd-demo@default
password: qvd-demo

- VM root login: There is no access as root within the images, but it you need to have root access within the image, you can execute the following commands in the server side as root:
# chroot /var/lib/qvd/storage/rootfs/10000-fs
# passwd
Enter new UNIX password: root
Retype new UNIX password: root
passwd: password updated successfully
# exit
# cd /

And when accessing to the VM with the QVD Client you can use the following credentials to log as root:
user: root
password: root

Hope this information helps you, if you have any other issue let us know.

Thanks for your patience.

**Comment by slacknet** [comment]
So. When I am importing the QVD-4.0.0-demo-server.ova with Virtual box is automaticlly pop into the command line and I am typing user qvd and passwd qvd, but if I want to input **"sudo su"** and I put the password root and it doesn't work. I know I mught missing something. Any link where I can see how to manage the demo installation, please?

**Comment by slacknet** [comment]
hi Fran. This Virtual Desktop is grate. There is possible to integrate it with Active Directory?

**Comment by slacknet** [comment]
I mean if I have Active Directory running on my company, t be able to import the users on the QVD system

**Comment by frantracer** [comment]
We have a solution to integrate QVD with Active Directory, but it is not included in the project. As you seem interested, our commercial team will get in touch in private with further information.

**Comment by slacknet** [comment]
Yes, off course. Let the commercial dept, to send me an email with the information ,pls

**Comment by slacknet** [comment]
Yes, off course. Let the commercial dept, to send me an email with the information ,pls

**Comment by slacknet** [comment]
Guys. What version of suse i need to use in order to install qvd?

**Comment by slacknet** [comment]
the version of suse on the installation guide is quite old

**Comment by frantracer** [comment]
We are running QVD in SLES12 SP1, you can check the quick installation guide as well that may help you:
http://theqvd.com/documentation/qvd-4-0-installation-server-sles

If you have any issue with SLES just post it here so we will try to help you.
